client/app/services/event-notifications.service.js

```diff
@@ -5,7 +5,7 @@
   .factory('EventNotifications', EventNotificationsFactory);
 
   /** @ngInject */
-  function EventNotificationsFactory($timeout, lodash) {
+  function EventNotificationsFactory($timeout, lodash, CollectionsApi, Session, logger) {
```

Comment:
Does `ActionCable` need to be added here?  (kinda rhetorical question, it does, but if it doesn't need to be to function, we need to declare it as a constant in a similar was as lodash and inject it into the factory)